cmd/estuary-shuttle/main.go

```diff
@@ -135,6 +135,27 @@ func main() {
 		&cli.BoolFlag{
 			Name: "private",
 		},
+		&cli.BoolFlag{
+			Name:  "jaeger-tracing",
+			Value: false,
+		},
+		&cli.StringFlag{
+			Name:  "jaeger-provider-host",
+			Value: "localhost",
+		},
+		&cli.IntFlag{
+			Name:  "jaeger-provider-port",
```

Comment:
would be nice to combine all these flags into one (or two, to set the sampler ratio separately)